typescript/agentkit/README.md

```diff
@@ -591,6 +594,39 @@ const walletData = await walletProvider.exportWallet();
 ```
 
 
+SVM:
+- [SolanaKeypairWalletProvider](./src/wallet-providers//solanaKeypairWalletProvider.ts)
+
+### SolanaKeypairWalletProvider
+
+The `SolanaKeypairWalletProvider` is a wallet provider that uses the API [Solana web3.js](https://solana-labs.github.io/solana-web3.js/).
+
+#### Solana Network Configuration
+
+The `SolanaKeypairWalletProvider` can be configured to use a specific network by passing the `networkId` parameter to the `fromNetwork` method. The `networkId` is the ID of the Solana network you want to use. Valid values are `solana-mainnet`, `solana-devnet` and `solana-testnet`.
```

Comment:
If we want to feel more Solana native, I think we should use the values `mainnet-beta`, `devnet` and `testnet`: https://solana-labs.github.io/solana-web3.js/types/Cluster.html

cc @John-peterson-coinbase

Reply:
these are network IDs specified by CDP for when CDP Wallet supports solana. This change will be larger than AgentKit

Reply:
+1 but will not land now